packages/flutter_adaptive_scaffold/lib/src/breakpoints.dart

```diff
@@ -344,4 +345,84 @@ class Breakpoint {
     }
     return currentBreakpoint;
   }
+
+  /// Returns true if the current platform is Desktop.
+  static bool isDesktop(BuildContext context) {
+    return Breakpoint.desktop.contains(Theme.of(context).platform);
+  }
+
+  /// Returns true if the current platform is Mobile.
+  static bool isMobile(BuildContext context) {
+    return Breakpoint.mobile.contains(Theme.of(context).platform);
+  }
+
+  // #docregion Breakpoint operators
+  /// Returns true if the current [Breakpoint] is greater than the given [Breakpoint].
```

Comment:
super nit: for all of these operators, maybe the language should be `Returns true if this [Breakpoint]....`. What do you think?

Reply:
Done